minecraft_servers/morland/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "morland",
+  "nice_name": "MorLand.cz",
+  "direct_ip": "play.morland.cz",
+  "server_wildcards": [
+    "%.morland.cz"
+  ],
+  "supported_languages": [
+    "cs",
+    "sk",
+    "en"
+  ],
+  "social": {
+    "web": "https://morland.cz",

```

Comment:
![Bildschirmfoto 2023-06-10 um 11 06 00](https://github.com/LabyMod/server-media/assets/45363287/0c7456ec-966e-4e0e-87de-cbc4a0844f7e)

Reply:
We got a blackout, we'll get it back on tonight.